docs/user-guide/story-setting.md

```diff
@@ -0,0 +1,19 @@
+ 
+ 
+For setting the information that appears on the story top bar on the *View Mode* the user need to go back on the *Edit Mode* and open the **Setting** button <img src="../img/button/setting-button.jpg" class="ms-docbutton"/>.
+
+<img src="../img/story-setting/story-setting.jpg" class="ms-docimage"/>
+
+The user can enable/disable the *Title*, the *Icon* and the *Navigation bar* and can :
+
+* Change the **Title**
+
+* Add an **Icon**. Could be an image with max 500 Kb and with not more of 48 px of hight and 96 px of width.
+
+* Select which sections will be visible on the *Navigation bar* by clicking on the name of that section. 
+
+<img src="../img/story-setting/setting-navabar.gif" class="ms-docimage" width="400px"/>
+
+Save the setting and go back to the *View Mode*, the top bar will look like this:
```

Comment:
Saving the story settings and going back in  *View Mode*, the top bar looks like this: